qa/tasks/kafka.py

```diff
@@ -70,6 +51,10 @@ def install_kafka(ctx, config):
                 args=['rm', '-rf', test_dir],
             )
 
+            ctx.cluster.only(client).run(
+                args=['rm', '-rf', '{tdir}/kafka-2.6.0-src.tgz'.format(tdir=teuthology.get_testdir(ctx))],
```

Comment:
it seems surprising to see a version hardcoded here

Reply:
Changed it and also working fine: http://pulpito.front.sepia.ceph.com/kapandya-2021-03-18_05:37:42-rgw:notifications-master-distro-basic-smithi/